backend/Pipfile

```diff
@@ -57,6 +58,8 @@ pandas = "*"
 drf-excel = "*"
 numpy = "*"
 coverage = "*"
+daphne = "*"
```

Comment:
Can you explain why we need `daphne` and `tatsu`, and why `tatsu` needs a pin?

Reply:
`daphne` is needed because `channels.testing` now requires it but doesn't install it as a subdependency (see [this](https://github.com/django/channels/issues/1942))

I think we should be able to get rid of the `tatsu` pin. I don't remember why I added it (could've been because they dropped support for Python 3.10, but this isn't relevant as we're using 3.11 anyway).

Reply:
I remember why I added `tatsu` now, it's because the latest Django base image uses Python 3.10. Will try and use an image with Python 3.11